frontend/src/metabase/common/components/DataPicker/types.ts

```diff
@@ -65,16 +65,15 @@ export type DataPickerValue =
   | ModelItem
   | MetricItem;
 
-export type NotebookDataPickerFolderItem =
-  | CollectionItem
-  | DatabaseItem
-  | SchemaItem;
+export type DataPickerFolderItem = CollectionItem | DatabaseItem | SchemaItem;
```

Comment:
The `Notebook` prefix didn't make sense (as this component is called `DataPicker`, not `NotebookDataPicker`) so I removed it.